components/add_users_to_group_modal/index.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+
+import {Action, ActionResult} from 'mattermost-redux/types/actions';
+
+import {GlobalState} from 'types/store';
+
+import {addUsersToGroup} from 'mattermost-redux/actions/groups';
+import {getGroup} from 'mattermost-redux/selectors/entities/groups';
+import {ModalData} from 'types/actions';
+import {openModal} from 'actions/views/modals';
+
+import AddUsersToGroupModal from './add_users_to_group_modal';
+
+type Actions = {
+    addUsersToGroup: (groupId: string, userIds: string[]) => Promise<ActionResult>;
+    openModal: <P>(modalData: ModalData<P>) => void;
+}
+
+type OwnProps = {
+    groupId: string;
+}
+
+function makeMapStateToProps() {
```

Comment:
You don't need the `makeMapStateToProps` you can just use `mapStateToProps` and get rid of the "return a function" approach. (you can see it in other `index.ts` files).

The point of `makeMapStateToProps` is to generate selectors at mount time that are reused and take advantage of the cache in runtime, if you are not generating that selectors, this is just more complex code without any need.